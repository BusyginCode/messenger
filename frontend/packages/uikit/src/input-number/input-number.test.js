import React from 'react'
import { shallow } from 'enzyme'
import InputNumber from './input-number'

describe('uikit/InputNumber', () => {
  it('should render correctly', () => {
    expect(shallow(<InputNumber value={1} />)).toMatchSnapshot()
  })
})
