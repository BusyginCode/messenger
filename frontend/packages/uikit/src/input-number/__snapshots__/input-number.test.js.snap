// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/InputNumber should render correctly 1`] = `
<Input
  intent="default"
  size="l"
  type="number"
  value={1}
/>
`;
